numpy/linalg/_linalg.py

```diff
@@ -2151,6 +2168,10 @@ def pinv(a, rcond=1e-15, hermitian=False):
 
     """
     a, wrap = _makearray(a)
+    if rtol is not None:
+        if rcond != 1e-15:
+            raise ValueError("`rtol` and `rcond` can't be both set.")
```

Comment:
What if someone sets `rcond=1e-15` and `rtol != None`. In that case `rtol` would be ignored. Of course that's a pretty unusual thing to happen.

Still, it might be better to change the default value of `rcond` to `None`, and if both are unset, assume the user meant `rcond=1e-15` to preserve backward compatibility. Also document that behavior.

Reply:
Or error if both are set to non-default values and leave the default alone.